collective/plonefinder/browser/finder.pt

```diff
@@ -72,7 +72,8 @@
                href="javascript: void(0);"
                onclick="Browser.maximize(); return false;">
             </a>
-            <div id="searchBrowserBox">
+            <div id="searchBrowserBox"
+                 tal:condition="view/allowsearch">
```

Comment:
I would rather name this `showsearch`
`allowsearch` gives the feeling it is about permission